### DISCUSSION TITLE
Seed data for development purposes

### DISCUSSION THREAD

**Discussion by ro-savage** [plausible/analytics#1298] [Anything Else]
I am currently looking at contributing to plausible.

However, just getting fake data in there was a bit of mission with things like remembering event as the same user, and getting different dates etc.

It would be great if there was a seed command that just seeded a month of random data, including UTMs, Google Search Terms, Browsers etc.

Then someone who is contributing could just run the single command, and have dummy data in there to play with from the start.

Adding a simple command like `dummy_event` that also accepts an object that contains data to overwrite the request (e.g. want to add utm links, or a different referrer) would also be quite nice.

Unfortunately I've never used erlang, so that side is a bit out of my depth.

**Comment by ukutaht** [comment]
Good point, thanks for bringing it up.

I've been using database dumps from our live demo for development purposes. I can provide them for you as well if you wish :)

I agree that ideally we would have a random seed process instead. Our live demo doesn't use all the features so it's not always sufficient for testing stuff.

**Reply by Mirlaaa** [reply]
Good Morning, @ukutaht! May you provide me some database dumps?

**Reply by ukutaht** [reply]
Sorry I've stopped this practice, we have a different setup now. You can run a script to generate seed data: https://github.com/plausible/analytics/blob/master/priv/repo/seeds.exs
